js/util/web_worker.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var Worker = require('../source/worker');
+
+module.exports = function () {
+    var parentListeners = [],
+        workerListeners = [],
+        parentBus = new MessageBus(workerListeners, parentListeners),
+        workerBus = new MessageBus(parentListeners, workerListeners);
+
+    parentBus.target = workerBus;
+    workerBus.target = parentBus;
+    // workerBus substitutes the WebWorker global `self`, and Worker uses
+    // self.importScripts for the 'load worker source' target.
+    workerBus.importScripts = function () {};
+
+    new Worker(workerBus); // eslint-disable-line no-new
```

Comment:
I'm stargint to think we disable this eslint rule altogether. We're working around it in a few other places.